.travis.yml

```diff
@@ -29,6 +29,8 @@ env:
   - PROVIDER=softlayer
   - PROVIDER=triton
 
+install: echo "Skipping install step, everything is included in the docker image"
```

Comment:
I think `true` + a comment might be the perfect solution :two_men_holding_hands: